metadata/nz_building_outlines_lifecycle.xml

```diff
@@ -1,331 +1,213 @@
+  <gmd:fileIdentifier>
+    <gco:CharacterString>ED6455C1-6BE8-4620-8BA8-EF14A193992A</gco:CharacterString>
+  </gmd:fileIdentifier>
+  <gmd:language>
+    <gco:CharacterString>eng</gco:CharacterString>
+  </gmd:language>
+  <gmd:characterSet>
+    <gmd:MD_CharacterSetCode codeList="http://asdd.ga.gov.au/asdd/profileinfo/gmxCodelists.xml#MD_CharacterSetCode" codeListValue="utf8">utf8</gmd:MD_CharacterSetCode>
+  </gmd:characterSet>
+  <gmd:hierarchyLevel>
+    <gmd:MD_ScopeCode codeList="http://asdd.ga.gov.au/asdd/profileinfo/GAScopeCodeList.xml#MD_ScopeCode" codeListValue="nonGeographicDataset">nonGeographicDataset</gmd:MD_ScopeCode>
+  </gmd:hierarchyLevel>
+  <gmd:hierarchyLevelName>
+    <gco:CharacterString>nonGeographicDataset</gco:CharacterString>
+  </gmd:hierarchyLevelName>
+  <gmd:contact>
+    <gmd:CI_ResponsibleParty>
+	
+	<gmd:individualName gco:nilReason="withheld"><gco:CharacterString></gco:CharacterString></gmd:individualName>
+	<gmd:organisationName><gco:CharacterString>LINZ - Land Information New Zealand</gco:CharacterString></gmd:organisationName>
+	<gmd:positionName><gco:CharacterString>Location Data Analyst</gco:CharacterString></gmd:positionName>
+  <gmd:contactInfo>
+        <gmd:CI_Contact>
+          <gmd:phone>
+            <gmd:CI_Telephone>
+              <gmd:voice>
+                <gco:CharacterString>04 4600110</gco:CharacterString>
+              </gmd:voice>
+            </gmd:CI_Telephone>
+          </gmd:phone>
+          <gmd:address>
+            <gmd:CI_Address>
+              <gmd:deliveryPoint>
+                <gco:CharacterString>155 The Terrace</gco:CharacterString>
+              </gmd:deliveryPoint>
+              <gmd:city>
+                <gco:CharacterString>Wellington</gco:CharacterString>
+              </gmd:city>
+              <gmd:postalCode>
+                <gco:CharacterString>6011</gco:CharacterString>
+              </gmd:postalCode>
+              <gmd:country>
+                <gco:CharacterString>New Zealand</gco:CharacterString>
+              </gmd:country>
+              <gmd:electronicMailAddress>
+                <gco:CharacterString>info@linz.govt.nz</gco:CharacterString>
+              </gmd:electronicMailAddress>
+            </gmd:CI_Address>
+          </gmd:address>
+        </gmd:CI_Contact>
+      </gmd:contactInfo>
+	
+<gmd:role><gmd:CI_RoleCode codeList="http://asdd.ga.gov.au/asdd/profileinfo/gmxCodelists.xml#CI_RoleCode" codeListValue="pointOfContact">pointOfContact</gmd:CI_RoleCode></gmd:role></gmd:CI_ResponsibleParty>
+</gmd:contact>
+  <gmd:dateStamp>
+    <gco:Date>2019-05-01</gco:Date>

```

Comment:
According to the ANZMet software, one of those is creation date, one is published date. All should be year/mon/day. I've changed this in all of them.

We created the lifecycle xml from the ANZMet software, whereas the other two we didn't.
It could be that since the lifecycle metadata is of type "nonGeographicDataset" and the buildings and buildings full history are of type "dataset" there could be these kinds of differences, but I'm not sure.